docs/vex.md

```diff
@@ -0,0 +1,234 @@
+# VEX Metadata From WolfiOS Packages 
+
+One of the key design features of the Wolfi toolchain is its rich generation
+of supply chain metadata as part of its build processes. All WolfiOS packages
+get a software bill of materials (SBOM) and to complement it, Vulnerability
+Exploitability Exchange (VEX) metadata can be generated to help with false
+positives in security scans exacerbated by the extreme transparency offered
+by all the available data.
+
+If you are unfamiliar with VEX, you can think of it as a negative security
+advisory. A document where software authors inform users that a vulnerability
+does not affect a program. For more information check out the VEX resources
+at Chainguard academy.
```

Comment:
This would be a good place for one or more links to more thorough explanations of VEX.

Reply:
I was planning to link to Academy, but I changed it to our blog post for now.